vignettes/creating_estimators.Rmd

```diff
@@ -441,7 +437,6 @@ model_fn <- function(features, labels, mode, params, config) {
   output_layer <- tf$layers$dense(second_hidden_layer, 1L)
   
   # Reshape output layer to 1-dim Tensor to return predictions
-  # TODO: This failed if it's c(-1L) - check in reticulate for single element vector conversion
```

Comment:
Expected behavior, length one vectors are mapped to scalars

Reply:
I just re-exported the `shape` function from tensorflow, which will enable us to write this as:

```r
predictions <- tf$reshape(output_layer, shape(-1))
```

Reply:
Should we discourage folks from writing things like `list(-1)` or `list(NULL, 3)` and encourage `shape()` across the board?

Reply:
This PR looks good to me, feel free to merge when ready!

Reply:
I am torn on this as `list` is a bit more transparent. 

BTW we should *never* require the `L` suffix in our R facing APIs. I realize in this case we are calling a `tf$` API which requires the `L` (so `shape` is handy because it does the as.integer).